app/screens/user_profile/user_profile.test.js

```diff
@@ -84,4 +87,70 @@ describe('user_profile', () => {
             />
         )).toEqual(true);
     });
+
+    test('should push EditProfile', async () => {
```

Comment:
Please unnecessary `async`, same case for other instances in this test.
```suggestion
    test('should push EditProfile', () => {
```